datalad/cmdline/helpers.py

```diff
@@ -335,41 +336,3 @@ def POC_get_root_handle(root_dir=None, path_only=False):
     from ..support.gitrepo import GitRepo
     return GitRepo(root_dir, create=True)
```

Comment:
The actual creation of the default root handle would be something for a setup/install script or sth like this - yes.